BUILD

```
@@ -190,7 +192,7 @@ NODEJS_VARIATIONS = [
 
 NODEJS = {
     "{REGISTRY}/{PROJECT_ID}/nodejs" + version + "-" + distro + ":" + tag_base + "-" + arch: "//nodejs:nodejs" + version + label + "_" + user + "_" + arch + "_" + distro
-    for arch in ARCHITECTURES
+    for arch in node_arch(version)
     for distro in DISTROS
     for version in NODEJS_VERSIONS
```

Likely this should do it:
```suggestion
    for version in NODEJS_VERSIONS
    for arch in node_arch(version)
    for distro in DISTROS
```